src/schemas/suggestion.ts

```diff
@@ -0,0 +1,49 @@
+import { sqliteTable, text, int } from "drizzle-orm/sqlite-core";
+import { sql, type InferInsertModel, type InferSelectModel } from "drizzle-orm";
+
+export const SUGGESTION_STATUS = {
+	POSTED: "POSTED",
+	ACCEPTED: "ACCEPTED",
+	REJECTED: "REJECTED",
+} as const satisfies Record<string, string>;
+
+export type SuggestionStatus =
+	typeof SUGGESTION_STATUS[keyof typeof SUGGESTION_STATUS];
+export type UpdatedSuggestionStatus = Exclude<SuggestionStatus, "POSTED">;
+
+const SUGGESTION_STATUS_VALUES = Object.values(SUGGESTION_STATUS) as [
+	"POSTED",
+	"ACCEPTED",
+	"REJECTED",
+];
+
+export const Suggestion = sqliteTable("suggestion", {
+	id: int("id").primaryKey({ autoIncrement: true }).notNull(),
+
+	title: text("title").notNull(),
+	description: text("description"),
+
+	guildId: text("guildId").notNull(),
```

Comment:
We should probably implement a foreign key constraint for this, as there should be any case where there is a suggestion attached to a server with no config set. We can also then use cascading deletes to ensure there are no orphaned suggestions.